core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -194,7 +194,7 @@ public OutputStream getOutStream(long blockId, long blockSize, WorkerNetAddress
     }
     // No specified location to write to.
     if (address == null) {
-      throw new RuntimeException(ExceptionMessage.NO_WORKER_AVAILABLE.getMessage());
+      throw new RuntimeException(ExceptionMessage.NO_SPACE_FOR_BLOCK_ON_WORKER.getMessage());
```

Comment:
pass `blockSize` to `getMessage()` so that it replaces the `{0}` in `NO_SPACE_FOR_BLOCK_ON_WORKER`